### PR TITLE
Swift graduates from school.

### PR DESCRIPTION
**Pull request by Sajjon**
I would argue that Swift is not *new* any more, it is a mature language at this point and the first sentence of `The Basics` of Swift ought to reflect that 🎓🎉

### CODE REVIEW

**Comment by Sajjon** [comment]
We might also want to consider changing `for iOS, macOS, watchOS, and tvOS app development.` to `all of Apples operating systems, Unix and Windows` or similar. It has been quite some time since Swift got Unix and Windows support, the Swift Programming Language book ought to reflect that :).

File: Sources/TSPL/TSPL.docc/LanguageGuide/TheBasics.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 # The Basics
 
-Swift is a new programming language for iOS, macOS, watchOS, and tvOS app development.
-Nonetheless, many parts of Swift will be familiar
-from your experience of developing in C and Objective-C.
+Swift is a programming language for iOS, macOS, watchOS, and tvOS app development.
+If you have prior programming experience in languages such as C or Objective-C,
+some parts of Swift will be familiar to you.
```
Comment (xwu) [inline Sources/TSPL/TSPL.docc/LanguageGuide/TheBasics.md:7]:
```suggestion
If you have experience developing in C or Objective-C,
many parts of Swift will be familiar to you.
```

As noted in the forum post, content-level changes will be solicited at a later time. To keep this PR focused on the proposed change (deleting "new"), the most faithful rewording of the sentence that follows would be something like this.

**Comment by buttaface** [comment]
> to all of Apples operating systems, Unix and Windows

I suggest listing out the Unix platforms that have actively maintained ports, ie linux, Android and OpenBSD, so as not to give the wrong impression that other platforms like FreeBSD or OpenSolaris are currently supported.

**Comment by Mordil** [comment]
> > to all of Apples operating systems, Unix and Windows
> 
> I suggest listing out the Unix platforms that have actively maintained ports, ie linux, Android and OpenBSD, so as not to give the wrong impression that other platforms like FreeBSD or OpenSolaris are currently supported.

This is ripe for going stale, so it might be good to give a coarse summary such as "Apple, Linux, and Windows platforms" with a link to the supported platforms page of Swift.org for people to dig into specifics.

**Review by amartini51 — APPROVED** [review]

**Comment by amartini51** [comment]
@swift-ci Please test.

**Pull request merged**